doc/elements-introduction.md

```diff
@@ -0,0 +1,67 @@
+
+The four specifications that together form Web Components are:
+
+- #### Template element
+  “The HTML template element `<template>` is a mechanism for holding client-side content that is not to be rendered when a page is loaded but may subsequently be instantiated during runtime using JavaScript.”
+
+- #### Shadow DOM
+  “Shadow DOM provides encapsulation for the JavaScript, CSS, and templating in a Web Component. Shadow DOM makes it so these things remain separate from the DOM of the main document.”
+
+  “Why would you want to keep some code separate from the rest of the page? One reason is that on a large site, for example, if the CSS is not carefully organized, the styling for the navigation can "leak" into the main content area where it was not intended to go, or vice-versa. As a site or an app scales, this kind of thing becomes difficult to avoid.”
+
+- #### HTML Imports
+  “HTML Imports is a way to include HTML documents in other HTML documents. You're not limited to markup either. An import can also include CSS, JavaScript, or anything else an .html file can contain. In other words, this makes imports a fantastic tool for loading related HTML/CSS/JS.”
+
+- #### Custom Elements
+  “Custom Elements allow web developers to define new types of HTML elements. It's quite possibly the most important as Web Components don't exist without the features unlocked by custom elements.”
+
+Descriptions provided by [MDN](https://developer.mozilla.org/en-US/) and [HTML5 Rocks](http://www.html5rocks.com/en/).
+
+[Learn more about Web Components »](http://webcomponents.org)
+
+### Polymer
+
+The Polymer library is designed to make it easier and faster for developers to create great, reusable components for the modern web.
+
+It provides a declarative syntax that makes it simpler to define custom elements. And it adds features like templating, two-way data binding and property observation to help you build powerful, reusable elements with less code.
+
+The Polymer team also builds and maintains a large collection of custom elements, and the most prominent one is the Paper collection. The Paper elements are Google’s reference implementation of Material Design for the web.
+
+[Learn more about Polymer »](https://www.polymer-project.org)
+
+[Browse Polymer Element Catalog »](https://elements.polymer-project.org)
+
+### Browser compatibility
+
+The standards of Web Components are on the cutting-edge of web technologies and as such they are not yet fully supported in all browsers. Vaadin elements rely on polyfills to fill the gap and can therefore support the latest versions of all major browsers.
+
+The current set of supported browsers for Vaadin Core Elements is listed on the table below.
+
+| IE / Edge | Firefox | Chrome | Safari | iOS Safari | Chrome Android |
+| :---------: | :---------: | :---------: | :---------: | :---------: | :---------: |
+| IE11, Edge| Latest | Latest | Latest | Latest | Latest
```

Comment:
We still support iOS 8, so that should be explicit in the table. Android might also be mentioned that it’s from 5.0 upwards.